### DISCUSSION TITLE
Seattle Future Land Use Plans 2035

### DISCUSSION THREAD

**Discussion by yodapancake** [jakobzhao/geog495#56] [Final Project]
Link to repository: https://github.com/yodapancake/geog495_finalproject
Link to application: https://yodapancake.github.io/geog495_finalproject/

## Project Description 
For my **Geog495** final project I built a webgis application that shows you the future land use plans for the city of Seattle that are to be implemented by 2035, and the different colors represent different zoning designations. I also included a search bar at the top for people to look up different locations and drop a pin on that location. The goal of my project is to show people how Seattle might be changing over the next few years and to help people see what might be happening to locations they are familiar with. For example, some parks are being removed or reduced in order to make space for more housing. One issue with the data is that sometimes cities designate zones with multiple designations, and this causes the polygons to overlap in the data which causes certain areas to become weird colors not in the defined color key because I color the polygons with an amount of opacity. To compensate for this, I also included borders for each zone so that people can easily distsniguish area from one anaother. In addition, if you click on any zone a box with information on the zone will pop up on the map and tell you what the area is designated for. 

![image](https://raw.githubusercontent.com/yodapancake/geog495_finalproject/main/img/overview.PNG)

**Comment by nlertv** [comment]
I think this is a very interesting project and it is definitely informative. I really like how you include a lot of different planned destinations of the land as well. I think the pop-up is also really helpful to the user since there is a lot of overlap on the map. One thing to improve would be to have the ability to filter out the different types of land use. I think this will help the user to see the specific region and the future land use even more clearly.

**Comment by emilyphantastic** [comment]
This is one of my favorite project I've seen. I love the idea and think it was executed amazingly. A suggestion I have is to make the text box on the top right corner bigger so I don't need to scroll to read what area I'm hovering over is.

**Comment by VkSath00** [comment]
This map is very well done and informative, I also like how colorful it is however there is a lot of information that is packed into this map and i believe it would be easier to read if various types of land use were filterable which would make the overall map a bit more easier to follow with specific land uses.

**Comment by madiframpton** [comment]
Hi! I love the concept of your application and your execution is great! I agree with the previous comments, the only thing I would like to see from this map is the capability to filter out different land uses so the user can choose what they want displayed. Really great work here!

**Comment by RensenJiang** [comment]
Hi, this is a great and informative map, the type of land use is clearly shown on the map. One thing that could make this better is to add a filter function so that users can have a better view.
